pkg/scheduler/plugins/util/framework.go

```diff
@@ -0,0 +1,86 @@
+package util
```

Comment:
let's move to `pkg/scheduler/plugins/util/k8s/framework.go`

Reply:
done